docs/data/install-xrootd.md

```diff
@@ -649,6 +709,87 @@ root@host # service frm\_xfrd start
 root@host # service frm\_purged start
 ```
 
+Installing a GridFTP Server
+---------------------------
+
+The Globus GridFTP server can be installed alongside an XRootD storage element to provide
+GridFTP-based access to the storage.
+
+!!! note "See Also"
+    OSG has extensive documentation on setting up a GridFTP server; this section is an
+    abbreviated version documenting the special steps needed for XRootD integration.
+    You may also find the following useful:
+
+    -   [Basic GridFTP Install](gridftp).  Additionally covers service planning topics.
+    -   [Load-balanced GridFTP Install](load-balanced-gridftp).  Covers the creation of
+        a load-balanced GridFTP service using multiple servers.
+
+### Engineering Considerations
```

Comment:
Since we have the note above about this being an abbreviated install guide, I don't think we need this section. Ditto for the "File location" and "services" sections